api/test/wfl/integration/modules/aou_test.clj

```diff
@@ -5,16 +5,19 @@
             [wfl.module.all :as all]
             [wfl.module.aou :as aou]
             [wfl.service.cromwell :refer [submit-workflow]]
-            [wfl.service.postgres :as postgres]
             [wfl.tools.fixtures :as fixtures]
             [wfl.tools.workloads :as workloads]
-            [wfl.tools.endpoints :as endpoints])
+            [wfl.tools.endpoints :as endpoints]
+            [wfl.service.cromwell :as cromwell])
   (:import (java.util UUID)))
 
 (use-fixtures :once fixtures/temporary-postgresql-database)
 
 (defn mock-submit-workload [& _] (UUID/randomUUID))
-(def mock-cromwell-status (constantly "Succeeded"))
+(defn mock-cromwell-query-id [_ params]
+  (->> params
+       (filter #(contains? % :id))
+       (map #(assoc % :status "Succeeded"))))
```

Comment:
Can `params` have `{… :id nil …}`? What would that mean?

Reply:
I suppose it would be possible to write code that would do that, but the `(remove (comp nil? :uuid))` we talked about above eliminates that case

Reply:
Then `#(contains? % :id)` is just `:id` unless `:id` maps to `nil`.